hudi-flink/src/main/java/org/apache/hudi/schema/FilebasedSchemaProvider.java

```diff
@@ -43,16 +45,13 @@
     private static final String TARGET_SCHEMA_FILE_PROP = "hoodie.deltastreamer.schemaprovider.target.schema.file";
```

Comment:
Will we always reuse the config option from `deltastreamer`? Let us think about it with more insight?

Reply:
We should keep the option for compatibility.

Reply:
Yes, but we can not always use a confused config option, here we are not `deltastreamer`. We can handle it later, can you file a jira issue to track it?

Reply:
Sure, see https://issues.apache.org/jira/browse/HUDI-1541